pkg/git/git_keychain.go

```diff
@@ -169,8 +181,9 @@ func (k *secretGitKeychain) Resolve(url string, username string, allowedTypes gi
 
 	for _, cred := range k.creds {
 		if cred.match(u.Host, allowedTypes) {
-			return cred.git2goCredential(username)
+			return cred.goGitCredential(username)
 		}
 	}
+
 	return nil, errors.Errorf("no credentials found for %s", url)
```

Comment:
Need to return anonymous auth a la https://github.com/pivotal/kpack/blob/d4de54b914365f6082bcbe938fa038aab18ed9b9/pkg/git/git_keychain.go#LL136C7-L136C7

Reply:
Done